### PR TITLE
Aman 86/add edit certif

### PR DESCRIPTION
**Pull request by Scarletra**
# Description  
Add new feature: add and edit certification

# Ticket Issue 
[[Provide link to Jira]  ](https://a-man-ppl.atlassian.net/browse/AMAN-86?atlOrigin=eyJpIjoiODg5NzkxOWYwZmRlNDM4N2FlMjc5YjM1MTM2MDM3NTgiLCJwIjoiaiJ9)

# Changes Made  
Add new button and modal on certification section

# Issue Type
- [ ] 🐛 Bug fix 
- [ ] ⚙️ Refactor
- [X] 💻 New feature 
- [ ] 📄 Documentation update
- [ ] ⚠️ Breaking changes
- [X] 🔍 Testing

# Added/updated tests?

- [X] Yes
- [ ] No, and this is why: [please replace this line with details on why tests have not been included]

# Additional Notes  
No extra notes

# (optional) What photo/gif best describes this PR or how it makes you feel?
None

## Summary by CodeRabbit

- **New Features**
	- Enhanced certificate management with an interactive modal for adding and editing entries, including publication date support and file upload functionality.
  
- **Style**
	- Updated visual presentation for location displays and user profile cards with refined typography, color adjustments, and improved layout aesthetics.
  
- **Tests**
	- Expanded quality assurance coverage to verify robust certificate interactions under various user scenarios, including positive, negative, and edge cases.

### CODE REVIEW

File: src/components/ui/certificate.tsx [hunk 1]
```diff
@@ -71,6 +218,67 @@ const Certificate: React.FC<CertificateProps> = ({ certificates = [] }) => {
                     Tidak ada sertifikasi.
                 </Typography>
             )}
+
+            {isModalOpen && (
+                <Modal
```
Comment (gnh374) [inline src/components/ui/certificate.tsx:223]: udah keren. tpi mungkin ini bisa dibuat komponen terpisah sendiri aja biar ga terlalu panjang

Comment (Scarletra) [reply src/components/ui/certificate.tsx:223]: ohh ini emng perlu direfact lagi sih tp mungkin bakal dijadiin task lain atau nanti ada pr terpisah, perlu untuk feat lain jadi dimerge dlu ya
